src/simulator_adapter.py

```diff
@@ -61,13 +61,18 @@ def sim_pose_callback(pose_msg):
         camera_pose_pub.publish(camera_pose_msg)
 
     if publish_ground_truth_altitude:
-        # TODO: Make this also publish the altimeter_reading topic
         # Publish altimeter_pose
         altimeter_pose = PoseWithCovarianceStamped()
         altimeter_pose.header.stamp = pose_msg.header.stamp
-        altimeter_pose.pose.position.z = pose_msg.pose.position.z
+        altimeter_pose.pose.pose.position.z = pose_msg.pose.position.z
         altimeter_pose_pub.publish(altimeter_pose)
 
+        # Publish altimeter raw height
+        altimeter_height = Float64Stamped()
+        altimeter_height.header.stamp = pose_msg.header.stamp
+        altimeter_height.data = pose_msg.pose.position.z
```

Comment:
It has to be Point(0, 0, -pose.position.z) in the level_quad frame transformed into the lidarlite frame